### ISSUE TITLE
can't import snowflake.ml.modeling.compose in windows 11 and python 3.9

### ISSUE THREAD

**Issue by albertsuzx** [snowflakedb/snowflake-ml-python#60]
I have followed the instruction to set up the environment but cannot import the following modules from snowflake-ml-python.
```
snowflake.ml.modeling.compose
snowflake.ml.modeling.preprocessing
snowflake.ml.modeling.linear_model
snowflake.ml.modeling.model_selection
```
I have got the error message below. And it looks like a issue with windows decoder cannot handle the internal code when I try to import the module itself.
```
SyntaxError: (unicode error) 'utf-8' codec can't decode byte 0x92 in position 5415: invalid start byte (column_transformer.py, line 246)
...
```

**Comment by sfc-gh-pdorairaj** [comment]
This is fixed in 1.0.10 release

**Comment by sfc-gh-pdorairaj** [comment]
I am closing it, pls reopen if it is still unresolved for you.
